app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -324,10 +326,15 @@ export default class TidyTree {
     }
 
     // adjust dimensions
-    let width = this.options.minWidth;
+    let width = Math.min(
+      Math.max(this.options.minWidth, this.options.width),
+      document.documentElement.clientWidth -
```

Comment:
We should adjust to the width of the parent container/div and not view port.
Specially because this is a generic tree. In other pages with elements left and right this might not work.